adapters/adhese/adhesetest/exemplary/banner-internal.json

```diff
@@ -0,0 +1,176 @@
+{
+  "mockBidRequest": {
+    "id": "test-req",
+    "imp": [
+      {
+        "id": "test-req",
+        "banner": {
+          "format": [
+            {
+              "w": 300,
+              "h": 250
+            }
+          ]
+        },
+        "ext": {
+          "bidder": {
+            "account": "demo",
+            "location": "_adhese_prebid_demo_",
+            "format": "leaderboard",
+            "targets": [
+              {
+                "key": "ci",
+                "value": [
+                  "gent",
+                  "brussels"
+                ]
+              },
+              {
+                "key": "ag",
+                "value": [
+                  "55"
+                ]
+              },
+              {
+                "key": "tl",
+                "value": [
+                  "all"
+                ]
+              }
+            ]
+          }
+        }
+      }
+    ],
+    "site": {
+      "id": "test",
+      "publisher": {
+        "id": "123"
+      }
+    }
+  },
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "https://ads-demo.adhese.com/json/sl_adhese_prebid_demo_-leaderboard/ag55/cigent;brussels/tlall?t=20200306",
+        "body": {
```

Comment:
This does not look legit.
What is this body? 
The body you expect the adapter to send to adhese? If so then that should be empty.
The body which is response of adhese? That should be our json.

Reply:
the response body is in 
`"mockResponse": {
        "status": 200,
        "body": [{`
right? So this should be empty?!

Reply:
Well sounds correct. If you do a GET /json request to our adserver, you don't send any body, do you? All the info is in the request itself. Also sending body for GET requests sounds a bit weird anyway (is it even correct according to http specs?).